calico/felix/config.py

```diff
@@ -184,6 +184,12 @@ def __init__(self, config_path):
                            "none", sources=[ENV, FILE])
         self.add_parameter("EtcdCaFile", "Path to etcd CA certificate file",
                            "none", sources=[ENV, FILE])
+        self.add_parameter("EtcdEndpoints", "Comma separated list of etcd "
```

Comment:
Code above has a separate `etcd_scheme` parameter, how do we square that with a list of endpoints that include the scheme?